.github/workflows/test.yml

```diff
@@ -13,28 +13,34 @@ jobs:
 
     steps:
     - name: 'Check out repository'
-      uses: actions/checkout@v2
+      uses: actions/checkout@v3
```

Comment:
Where do I find the definition of this action?

Reply:
It is here: https://github.com/actions/checkout

The common pattern is to prepend `https://github.com/` to an action in use - and omit the version information.

Reply:
Thank you